### PR TITLE
Issue #136 Update README.md

### PR DESCRIPTION
**Pull request by tsujiguchitky**
## Analysis

We forked ForgeRock OpenAM as OpenAM Consortium edition. 
However, our README.md indicates ForgeRock projects. We should resolve this before a new release of OpenAM Consortium edition.

## Solution

Make full review of README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,230 +1,73 @@
...
-The OpenAM build process and dependencies are managed by Maven. The first time you build the project, Maven will pull 
-down all the dependencies and Maven plugins required by the build, which can take a significant amount of time. 
-Subsequent builds will be much faster!
+At this time, OpenAM and related products are not registered in the Maven repository, so it is necessary to build all projects until the Maven local repository is ready.
 
-```
-cd openam
-mvn clean install
-```
+Please, check out the following projects and run `mvn install` in order.
```
[the remark this answers is not in the document]
Comment (tsujiguchitky) [reply README.md]: Thank you for the review. 

The points you pointed out are certainly true. I will fix.

File: README.md [hunk 2]
```diff
@@ -1,230 +1,73 @@
...
 ## Contributing
 
-There are many ways to contribute to the OpenAM project. You can contribute to the [OpenAM Docs Project][docs_project], 
-report or [submit bug fixes][issue_tracking], or [contribute extensions][contribute] such as custom authentication 
-modules, authentication scripts, policy scripts, dev ops scripts, and more.
+This project welcomes your contributions.
 
-## Versioning
+If you find a bug or have an improvement idea, please open an [issue][github_issues] as the first step.
+However, if it contains security problems, please contact [us][mail_openam_dev] without opening an issue.
```
[the remark this answers is not in the document]
Comment (tsujiguchitky) [reply README.md]: make sense.

File: README_ja.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* [openam](https://github.com/openam-jp/openam)
+
+最後に、`openam/openam-server/target` にバイナリが作成されます。ファイル名の形式は `OpenAM-<バージョン>.war` です。
+
+## コントリビューション
+
+このプロジェクトはあなたのコントリビューションを歓迎します。
+
+バグを見つけたり、改善のアイデアをお持ちの場合は、最初のステップとして [Issue][github_issues] をオープンしてください。
+ただし、セキュリティ上の問題が含まれている場合は、[Issue][github_issues] をオープンせずに[ご連絡ください][mail_openam_dev]。
```
[the remark this answers is not in the document]
Comment (tsujiguchitky) [reply README_ja.md]: なるほど、承知しました。

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ogis-miyamura) [reply README.md]: Thanks for fixing.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ogis-miyamura) [reply README.md]: Thank you for considering.

File: README_ja.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ogis-miyamura) [reply README_ja.md]: 検討していただきありがとうございます。

**Pull request merged**